### ISSUE TITLE
Start with user needs

### ISSUE THREAD

**Issue by pmackay** [ouisharelabs/food-dashboard#3]
Would be good to try to list some specific user needs that this would aim to solve. What benefits are gained by aggregating?

**Comment by wvengen** [comment]
Some needs that I find around me:
- As someone who is interested to learn more about food-related cooperative activities in my neighbourhood, I'd like to find out where I can go to.
- As a member of a food collective, I would like to find opportunities for cooperation with other groups, to enlarge our impact and solve problems that we can't address with just our collective (cooperative sourcing, for example).
- As a member of a food collective, I'd like to reduce the tedious work. Importing and editing product lists, would be a concrete example.
- As a person looking for fair, sustainable and affordable food supplies, I'd like find where I can get it from.
- As someone interested in buying local produce, I'd like to discover local producers, find what products they have, what the availability is, and how much it would cost.
- As a farmer interested in supplying to people directly, I'd like to find existing groups to deliver to.
- As a developer of foodcoop software, I'd like to see how I can contribute best to make a change.
- As an eater, I'd like to know what I'm eating, and what effect my choice has on the world.
- When doing groceries, I'd like to make an informed choice.
- As a member of society, I'd like to support grass-roots initiatives for a better world with regards to food.
- etc.

These may or may not be addressed by this initiative, and there are sure many more needs.
Hope this helps to get the ideas flowing.

**Comment by pmackay** [comment]
@wvengen those sound great! @elf-pavlik would you have any others? 

Would it be worth turning some of these into issues and discussing in more detail?

**Comment by elf-pavlik** [comment]
> As someone who is interested to learn more about food-related cooperative activities in my neighbourhood, I'd like to find out where I can go to.

This one relates directly to first step which I propose, aggregated map and calendar so people know where and when find what they look for. On a map side I plan to discuss it with friends working on http://transformap.co/ very soon.

**Comment by pmackay** [comment]
@elf-pavlik is that just about ensuring that the various food activities these orgs know about are mapped via Transformap?

**Comment by elf-pavlik** [comment]
No, I hope all the data about locations (farms, hubs etc.) and events (distribution events, volunteering events) get published in common way and one can aggregate easily.
Transformap can act as just one of various aggregation services!
